internal/controller/metalstackmachine_controller.go

```diff
@@ -144,9 +143,16 @@ func (r *MetalStackMachineReconciler) Reconcile(ctx context.Context, req ctrl.Re
 	var result ctrl.Result
 
 	if annotations.IsPaused(cluster, infraMachine) {
-		conditions.MarkTrue(infraMachine, v1alpha1.ProviderMachinePaused)
+		conditions.Set(infraMachine, metav1.Condition{
+			Status: metav1.ConditionTrue,
+			Type:   clusterv1.PausedCondition,
+		})
 	} else {
-		conditions.MarkFalse(infraMachine, v1alpha1.ProviderMachinePaused, clusterv1.PausedV1Beta2Reason, clusterv1.ConditionSeverityInfo, "")
+		conditions.Set(infraMachine, metav1.Condition{
+			Status: metav1.ConditionFalse,
+			Type:   clusterv1.PausedCondition,
+			Reason: clusterv1.PausedReason,
```

Comment:
Here only one paused condition has a reason, for metalstackclusters both have. How is this solved by official CAPI resources?